cypress/integration/edit-post.test.ts

```diff
@@ -0,0 +1,126 @@
+import { HttpMethod } from "@lib/http-method";
+import { StatusCodes } from "http-status-codes";
+import { TEST_POST_2_UUID } from "../support/constants";
+
+describe("Edit post", () => {
+  beforeEach(() => {
+    cy.task("db:setup");
+  });
+
+  afterEach(() => {
+    cy.task("db:teardown");
+  });
+
+  it("should allow a user edit a post from the account page", () => {
+    cy.login();
+    cy.visit("/account");
+
+    cy.intercept(HttpMethod.PUT, "/api/post/*").as("editPost");
+    cy.intercept(HttpMethod.GET, "/api/user/*").as("getUserPosts");
+
+    // Check if editing the post works
+    cy.findAllByRole("button", { name: /edit post/i })
+      .eq(0)
+      .click();
+
+    cy.findByRole("dialog")
+      .get('[id="description"]')
+      .click()
+      .type("EDITING THE DESCRIPTION");
+
+    cy.findByRole("dialog")
+      .get('[id="price"]')
+      .click()
+      .type("{backspace}{backspace}{backspace}{backspace}13");
```

Comment:
`{selectall}` instead of four `{backspace}` might be a bit more resilient to change

Reply:
Smart! Will update it.